src/activities/Elsa.Activities.RabbitMq/Services/Client.cs

```diff
@@ -53,7 +53,7 @@ public async Task PublishMessage(string message)
         {
             if (_bus == null) ConfigureAsOneWayClient();
 
-            await _bus.Advanced.Topics.Publish(Configuration.RoutingKey, message, Configuration.Headers);
+            await _bus!.Advanced.Topics.Publish(Configuration.RoutingKey, message, Configuration.Headers);
```

Comment:
We shouldn't have to use the exclamation mark here since there is a null check on line 54.